src/database/migrations/20200928102122-create-user.js

```diff
@@ -22,6 +22,30 @@ module.exports = {
       password: {
         type: Sequelize.STRING
       },
+      gender: {
+        type: Sequelize.STRING
+      },
+      birthdate: {
+        type: Sequelize.DATE
+      },
+      language: {
+        type: Sequelize.STRING
+      },
+      currency: {
+        type: Sequelize.STRING
+      },
+      country: {
+        type: Sequelize.STRING
+      },
+      role: {
+        type: Sequelize.STRING
+      },
+      department: {
+        type: Sequelize.STRING
+      },
```

Comment:
Hey, remember to not touch old migrations or rename them. You have to create new migrations for each feature.